src/vnc.yml

```
@@ -96,6 +110,3 @@
   vars:
     # The username, password, and ssh keys for the VNC user
```

Update comment to reflect variables that moved to earlier in this file:
```suggestion
    # The username for the VNC user
```